pyrdp/mitm/DeviceRedirectionMITM.py

```diff
@@ -70,19 +70,21 @@ def __init__(self, client: DeviceRedirectionLayer, server: DeviceRedirectionLaye
         self.state = state
         self.log = log
         self.statCounter = statCounter
-        self.config = config
         self.currentIORequests: Dict[int, DeviceIORequestPDU] = {}
         self.openedFiles: Dict[int, FileProxy] = {}
         self.openedMappings: Dict[int, FileMapping] = {}
         self.fileMap: Dict[str, FileMapping] = {}
         self.fileMapPath = self.config.outDir / "mapping.json"
         self.forgedRequests: Dict[int, DeviceRedirectionMITM.ForgedRequest] = {}
 
-        self.responseHandlers: Dict[MajorFunction, callable] = {
-            MajorFunction.IRP_MJ_CREATE: self.handleCreateResponse,
-            MajorFunction.IRP_MJ_READ: self.handleReadResponse,
-            MajorFunction.IRP_MJ_CLOSE: self.handleCloseResponse,
-        }
+        if state.config.extractFiles:
```

Comment:
The `config()` `@property` was not used here to access the config. Any reason why? Maybe because it is `__init__` and it wouldn't work?

Reply:
Mostly because  the state object is available directly, but I'll change it to be consistent.